armada/conf/default.py

```diff
@@ -17,6 +17,17 @@
 from oslo_config import cfg
 
 default_options = [
+
+    cfg.ListOpt(
+        'armada_apply_roles',
+        default=['admin'],
+        help='IDs of approved API access roles.'),
+
+    cfg.StrOpt(
```

Comment:
After discussing with @gardlt, we will create an issue for this suggestion and iterate on it